examples/overlay/messages/09_input_jaccard.json

```diff
@@ -0,0 +1,88 @@
+{
+    "query_graph": {
+        "nodes": {
+            "type-2 diabetes": {
+                "id": "MONDO:0005148"
+            },
+            "protein": {
+                "category": "biolink:Protein"
+            },
+            "drug": {
+                "category": "biolink:ChemicalSubstance"
+            }
+        },
+        "edges": {
+            "correlated": {
+                "subject": "protein",
+                "predicate": "biolink:correlated_with",
+                "object": "type-2 diabetes"
+            },
+            "interacts": {
+                "subject": "drug",
+                "predicate": "biolink:physically_interacts_with",
+                "object": "protein"
+            }
+        }
+    },
+    "knowledge_graph": {
+        "nodes": {
+            "MONDO:0005148": {"name": "type-2 diabetes"},
+            "CHEBI:6801": {
+                "name": "metformin",
+                "category": "biolink:ChemicalSubstance"
+            },
+            "CHEBI:5441": {
+                "name": "glyburide",
+                "category": "biolink:ChemicalSubstance"
+            },
+            "UniProtKB:Q92786": {
+                "name": "PROX1",
+                "category": "biolink:Protein"
+            },
+            "UniProtKB:P30279": {
+                "name": "CCND2",
+                "category": "biolink:Protein"
+            }
+        },
+        "edges": {
+            "k7g89spc": {
+                "subject": "CHEBI:6801",
+                "predicate": "biolink:physically_interacts_with",
+                "object": "UniProtKB:P30279"
+            },
+            "df87ff82": {
+                "subject": "CHEBI:6801",
+                "predicate": "biolink:physically_interacts_with",
+                "object": "UniProtKB:Q92786"
+            },
+            "5133c100": {
+                "subject": "CHEBI:5441",
+                "predicate": "biolink:physically_interacts_with",
+                "object": "UniProtKB:P30279"
+            },
+            "vk456hb1": {
+                "subject": "UniProtKB:Q92786",
+                "predicate": "biolink:correlated_with",
+                "object": "MONDO:0005148"
+            },
+            "lkg89002": {
+                "subject": "UniProtKB:P30279",
+                "predicate": "biolink:correlated_with",
+                "object": "MONDO:0005148"
+            }
+        }
+    },
+    "results": [
+        {
+            "node_bindings": {
+                "type-2 diabetes": [{"id": "MONDO:0005148"}],
+                "drug": [{"id": "CHEBI:6801"}],
+                "protein": [{"id": "UniProtKB:Q92786"},{"id": "UniProtKB:P30279"}]
```

Comment:
This is a set? Is that necessary for the Jaccard overlay to work? Theoretically, we could do this just as well if those two proteins were in separate results, or in no results at all. What happens if there are other proteins in the knowledge graph that are connected to these end nodes but are not bound in this result?

Reply:
Correct me if I am wrong here @dkoslicki but I believe that additional proteins in the knowledge graph even if not included in the results would change the jaccard values. 

I think this was what I was originally going to do but then changed it and forgot to change the values. Currently I think the values should be 1.0 and ,5 but if we added a 3rd protein in that was not in the result set but in the knowledge graph it would actually be .66 and .33.

Reply:
@finnagin Right: the Jaccard value is computed from the knowledge graph, not the results. But it shouldn't need `is_set=True` to work.